multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -285,3 +317,60 @@ func (r *MemberClusterAnnounceReconciler) GetMemberClusterStatuses() []multiclus
 
 	return status
 }
+
+func (r *MemberClusterAnnounceReconciler) addMemberToClusterSet(memberClusterAnnounce *multiclusterv1alpha1.MemberClusterAnnounce) error {
+	clusterSetID := memberClusterAnnounce.ClusterSetID
+	clusterSet := &multiclusterv1alpha1.ClusterSet{}
+	if err := r.Get(context.TODO(), types.NamespacedName{Namespace: memberClusterAnnounce.Namespace, Name: clusterSetID}, clusterSet); err != nil {
+		klog.ErrorS(err, "Failed to get ClusterSet in leader cluster", "ClusterSet", clusterSetID)
+		return err
+	}
+
+	exist := false
+	for _, member := range clusterSet.Spec.Members {
+		if member.ClusterID == memberClusterAnnounce.ClusterID {
+			exist = true
+			break
+		}
+	}
+
+	if exist {
```

Comment:
Please add a comment about "ClusterSet ID of MemberClusterAnnounce cannot change. It is guaranteed by the MemberClusterAnnounce webhook."

Reply:
Comment added.